br/Makefile

```diff
@@ -118,3 +118,22 @@ clean:
 	rm -rf *.out
 	rm -rf bin
 	rm -rf tools/bin
+
+br_integration_test: br_bins build_br build_for_br_integration_test
+	@cd br && tests/run.sh
```

Comment:
1. would it raise an error if we build the target `br_integration_test`?
2. why name it with `br_` prefix?
3. how about adding this integration test to [.github/workflow/ci-br.yml](https://github.com/tikv/migration/blob/main/.github/workflows/ci-br.yml)?